0009-jetstream.md

```diff
@@ -0,0 +1,79 @@
+# Use JetStream for Distributed Lattice Cache
+
+* Status: _Accepted_
+
+## Context and Problem Statement
+
+During the course of normal operation, wasmCloud hosts need to share certain information; metadata about the contents and operation of the _lattice_. This includes information such as a lookup table of JWT claims, a lookup table that maps OCI URLs to public keys, and a list of link definitions.
+
+This information needs to be shared with all hosts in a lattice at all times, and it needs to withstand things like partition events and temporary connectivity loss, as well as expected events like the termination and starting of individual host processes.
+
+The current solution to this problem is not reliable, doesn't scale beyond simple installations and, worse, can quite easily result in "split-brain" problems where different hosts have different views of the same set of information, potentially causing runtime failures in production.
+
+## Decision Drivers 
+
+* Reliability
+* Developer Experience
+* Predictability
+* Scalability
+* Resiliency
+* Enterprise suitability
+
+## Considered Options
+
+* Leave As-Is
+* Use Redis
+* Use memcached/etcd/consul or their ilk
+* Use NATS JetStream
+
+## Decision Outcome
+
+Chosen option: _**Use NATS JetStream**_, because this is the only option that actually satisfies all of our runtime production requirements while still enabling a simple developer experience and _not_ burdening the developer by requiring the installation of additional software.
+
+### Positive Consequences 
+
+* Gain a "free" external source of truth for the lattice cache without requiring developers to install additional products
+* Gain access to all of the JetStream features, including defining replication, clusters, persistence, and more
+* Ability to provide a reasonable default while allowing organizations and enterprises to define their own stream configuration
+
+### Negative Consequences
+
+* Requires developers to enable the JetStream feature when starting NATs servers for development (goes to developer experience)
```

Comment:
_Massive_ nitpick here but I think `s/NATs/NATS/` is worth doing, just to be extra clear we aren't talking about network address translation